tests/5.2/declare_target/test_declare_target_enter_device_type_any.c

```diff
@@ -0,0 +1,75 @@
+//===---- test_declare_target_enter_device_type_any.c -------------------------------------===//
+// 
+// OpenMP API Version 5.2 
+//
+// The enter clause was added as a synonym for the to clause on the declare target directive.
+// The device_type clause specifies if a version of the procedure or variable should be made
+//  available. If any is specified then both host device and non-host device versions of the 
+// procedure or variable are made available.
+
+//===-------------------------------------------------------------------------------===//
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+#define N 1024
+int errors = 0;
+
+#pragma omp declare target 
+int a[N], b[N], c[N];  
+int i = 0;
+#pragma omp end declare target
+
+void update() { 
+  for (i = 0; i < N; i++) {
+    a[i] += 1;
+    b[i] += 2;
+    c[i] += 3;
+  }
+}
+
+#pragma omp declare target enter(update) device_type(any) 
+
+int test_declare_target_device_type_any() { 
```

Comment:
GCC warns:
```
test_declare_target_enter_device_type_any.c:72:38: warning: implicit declaration of function ‘test_declare_target_enter_device_type_any’; did you mean ‘test_declare_target_device_type_any’? [-Wimplicit-function-declaration]
   72 |   OMPVV_TEST_AND_SET_VERBOSE(errors, test_declare_target_enter_device_type_any());
```
Thus, I suggest the following. With it, it passes (only host fallback tried). Otherwise, the same remark applies as above.
```suggestion
int test_declare_target_enter_device_type_any() { 
```